src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Server/Controllers/WeatherForecastController.cs

```diff
@@ -1,128 +1,120 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-#if (!NoAuth)
+﻿#if (!NoAuth)
 using Microsoft.AspNetCore.Authorization;
 #endif
 #if (GenerateApi)
-using Microsoft.Extensions.Configuration;
 using Microsoft.Identity.Web;
 using System.Net;
-using System.Net.Http;
 #endif
 #if (GenerateGraph)
 using Microsoft.Graph;
 #endif
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
 #if (OrganizationalAuth || IndividualB2CAuth)
 using Microsoft.Identity.Web.Resource;
 #endif
 using ComponentsWebAssembly_CSharp.Shared;
 
-namespace ComponentsWebAssembly_CSharp.Server.Controllers
-{
+namespace ComponentsWebAssembly_CSharp.Server.Controllers;
+
 #if (!NoAuth)
-    [Authorize]
+[Authorize]
 #endif
-    [ApiController]
-    [Route("[controller]")]
-    public class WeatherForecastController : ControllerBase
+[ApiController]
+[Route("[controller]")]
+public class WeatherForecastController : ControllerBase
+{
+    private static readonly string[] Summaries = new[]
     {
-        private static readonly string[] Summaries = new[]
-        {
-            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
-        };
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+    };
 
-        private readonly ILogger<WeatherForecastController> _logger;
+    private readonly ILogger<WeatherForecastController> _logger;
 #if (OrganizationalAuth || IndividualB2CAuth)
 
-        // The Web API will only accept tokens 1) for users, and 2) having the "api-scope" scope for this API
-        static readonly string[] scopeRequiredByApi = new string[] { "api-scope" };
+    // The Web API will only accept tokens 1) for users, and 2) having the "api-scope" scope for this API
+    static readonly string[] scopeRequiredByApi = new string[] { "api-scope" };
 #endif
 
 #if (GenerateApi)
-        private readonly IDownstreamWebApi _downstreamWebApi;
+    private readonly IDownstreamWebApi _downstreamWebApi;
 
-        public WeatherForecastController(ILogger<WeatherForecastController> logger,
-                              IDownstreamWebApi downstreamWebApi)
+    public WeatherForecastController(ILogger<WeatherForecastController> logger,
+                            IDownstreamWebApi downstreamWebApi)
+    {
+            _logger = logger;
+        _downstreamWebApi = downstreamWebApi;
+    }
+
+    [HttpGet]
+    public async Task<IEnumerable<WeatherForecast>> Get()
+    {
+        HttpContext.VerifyUserHasAnyAcceptedScope(scopeRequiredByApi);
+
+        using var response = await _downstreamWebApi.CallWebApiForUserAsync("DownstreamApi").ConfigureAwait(false);
```

Comment:
@davidfowl does this trigger you?

Reply:
For the record, I didn't change that, that's simply what was there already 🙃

Reply:
What the *&^%%$#@ is this

Reply:
Why is it using configure await, I have so many questions.

Reply:
This was part of the Identity.Web work: https://github.com/dotnet/aspnetcore/issues/24024